docs/installing-providers-from-sources.rst

```diff
@@ -0,0 +1,116 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Installing Provider from Sources
+--------------------------------
+
+.. contents:: :local:
+
+
+Installing Provider from released sources and packages
+''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+You can also install the provider using the official sources and packages. Those sources and packages
+released are the "official" sources of installation that you can use if you want to verify the
+origin of the packages and want to verify checksums and signatures of the packages.
+
+The packages are available at the
+`Official Apache Software Foundations Downloads page <https://downloads.apache.org/airflow/providers>`_
+
+The downloads are available at:
+
+.. jinja:: official_download_page
+
+    * `Sdist package <{{ closer_lua_url }}/{{ package_name }}-{{ package_version }}.tar.gz>`__ (`asc <{{ base_url }}/{{ package_name }}-{{ package_version }}.tar.gz.asc>`__, `sha512 <{{ base_url }}/{{ package_name }}-{{ package_version }}.tar.gz.sha512>`__) - those are also official sources for the package
+    * `Wheel package <{{ closer_lua_url }}/{{ package_name_underscores }}-{{ package_version }}-py3-none-any.whl>`__ (`asc <{{ base_url }}/{{ package_name_underscores }}-{{ package_version }}-py3-none-any.whl.asc>`__, `sha512 <{{ base_url }}/{{ package_name_underscores }}-{{ package_version }}-py3-none-any.whl.sha512>`__)
+
+If you want to install from the source code, you can download from the sources link above, it will contain
+a ``INSTALL`` file containing details on how you can build and install the provider.
+
+Release integrity & Verification of releases
+''''''''''''''''''''''''''''''''''''''''''''
+
+`PGP signatures KEYS <https://downloads.apache.org/airflow/KEYS>`_
+
+It is essential that you verify the integrity of the downloaded files using the PGP or SHA signatures.
+The PGP signatures can be verified using GPG or PGP. Please download the KEYS as well as the asc
+signature files for relevant distribution. It is recommended to get these files from the
+main distribution directory and not from the mirrors.
+
+.. code-block:: bash
+
+    gpg -i KEYS
+
+    or
+
+    pgpk -a KEYS
+
+    or
+
+    pgp -ka KEYS
```

Comment:
A nit, but what would you say to remove `or` and add note before code block with something like "Depending on your cli tool run one of the following"?

Reply:
I think it was just copied from ASF instructions: http://ws.apache.org/mirrors.cgi :)

Reply:
On that page the or is outside code block with imho is more visually pleasant

Reply:
Very good point :). Fixed

Reply:
@kaxil  - you might want to fix it for "airflow" page too :)

Reply:
![Screenshot 2021-09-12 21 38 10](https://user-images.githubusercontent.com/595491/133000515-c0b863b5-3020-420f-8582-8d40f319d7dd.png)